scripts/migration_scripts/SendtoplaylistToChartData.py

```
@@ -0,0 +1,169 @@
+import os
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)